src/types/types.ts

```diff
@@ -351,3 +351,35 @@ export interface SavedRpcMethod extends LocalStorageSavedItem {
   shareableUrl: string | undefined;
   payload: AnyObject;
 }
+
+// =============================================================================
+// Smart Contract Explorer
+// =============================================================================
+export type ContractInfoApiResponse = {
+  contract: string;
+  created: number;
+  creator: string;
+  account?: string;
+  payments?: number;
+  trades?: number;
+  wasm?: string;
+  storage_entries?: number;
+  validation?: {
+    status?: "verified" | "unverified";
+    repository?: string;
+    commit?: string;
+    package?: string;
+    make?: string;
+    ts?: number;
+  };
+  versions?: number;
+  salt?: string;
+  asset?: string;
+  code?: string;
+  issuer?: string;
+  functions?: {
```

Comment:
i don't see `functions` from contracts-api.yml's ContractInfo schema

Reply:
`functions` is returned in the API response but not documented. I tried to combine both versions for types.
![image](https://github.com/user-attachments/assets/959d6225-c2d8-483c-81d5-9203dcd5b531)